HotelPriceScout/Pages/Component.razor

```diff
@@ -0,0 +1,48 @@
+﻿@page "/vaterpas"
+<h3>Price thermometer</h3>
+
+<div class="row border col-md-4"> <!-- outer layer-->
+
+    @if(Test == null){<div><p>Loading data</p> </div>}// checks if thermometer is ready yet
+    else{
+
+        @foreach (Prices item in Test)
```

Comment:
Vi skal huske at flette det sammen med det pris-objekt, som Sarmi og jeg har lavet.

Reply:
What you want me do?